src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 	wg.Add(2)
 	go assistedController.ApproveCsrs(done, &wg)
 	go assistedController.PostInstallConfigs(&wg)
+	go assistedController.UpdateBMHs(&wg)
```

Comment:
don't we need to delete 	go assistedController.ApproveCsrs(done, &wg)
 ?

Reply:
i thought to delete it, but now there is a talk of maybe removing BMHs at all for 4.6 ( in a later commits), so i will leave it here for now